gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -137,23 +152,23 @@ def compute_inlier_mask(
 
         for (i2, i1) in outlier_weights_sum:
             if outlier_weights_sum[(i2, i1)] / len(projection_directions) < OUTLIER_WEIGHT_THRESHOLD:
-                inlier_idxs.add((i1, i2))
+                inliers.add((i1, i2))
 
-        return inlier_idxs
+        return inliers
 
     def _get_prior_measurements_in_world_frame(
         self,
         relative_pose_priors: Dict[Tuple[int, int], PosePrior],
```

Comment:
Can we rename it to `i1Ti2_priors`?

Reply:
Now renamed to `i2Ti1_priors`, as it looks like we are pulling out `i2Ti1` from the dictionary.